frontend/src/css/imports.css

```diff
@@ -7736,9 +7737,16 @@ fieldset[disabled] .worksheet_content .header-row .controls .edit-features .btn-
   }
   #ws_search,
   #worksheet .ws-panel,
-  #worksheet .dragbar {
+  #terminal-button {
```

Comment:
We should really get off of this css file soon before we start hacking this too much...

Reply:
I see. In terms of hiding CLI when the screen is smaller than a certain size, is that expected behavior? Should I remove this restriction completely? 

Right now what I'm doing here is to hide the Open/Hide Terminal toggle bar when the screen becomes too small.